src/lib/formValidator.ts

```diff
@@ -0,0 +1,19 @@
+import { FieldValidator } from "./fieldValidator";
+
+export class FormValidator {
+  private _fields: FieldValidator[];
+
+  constructor(...fields: FieldValidator[]) {
+    this._fields = fields;
+  }
+
+  hasError(): boolean {
```

Comment:
Looks like it's moving in the right direction 👍  well done mate!

One thing to note is that you can add a `public` identifier to the methods on this class that are public, as well as the `FieldValidator`, i.e:

````typescript
public hasError(): boolean {
...
}
````